### PR TITLE
Tobias 0.5.0 fix for build error

### PR DESCRIPTION
**Pull request by msbentsen**
:information_source:
If this is your first pull request to Bioconda please ping `@bioconda/core` so it can be reviewed and to request being added as a member of the Bioconda team. Members of the Bioconda team are able to merge their own pull requests once tests and linting have passed.

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by jenzopr** [comment]
Build is successful, however error is `pkg_resources.DistributionNotFound: The 'adjustText' distribution was not found and is required by tobias` during testing. Maybe add `adjusttext` to the run section?

**Comment by msbentsen** [comment]
> 
> 
> Build is successful, however error is `pkg_resources.DistributionNotFound: The 'adjustText' distribution was not found and is required by tobias` during testing. Maybe add `adjusttext` to the run section?

I would, but adjusttext is not available from bioconda/conda-forge, only from https://anaconda.org/phlya/adjusttext. Can I add conda packages from other channels as dependencies?

**Comment by jenzopr** [comment]
Is it not this one: https://github.com/conda-forge/adjusttext-feedstock ?

**Comment by jenzopr** [comment]
@BiocondaBot recheck

**Comment by jenzopr** [comment]
@bioconda-bot rebuild

**Comment by jenzopr** [comment]
@bioconda-bot rebuild

**Comment by jenzopr** [comment]
@bioconda-bot merge

File: recipes/tobias/meta.yaml [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 7c7de1efc97352b6fb7bc21a692c901dc7a8ab270a850445577ae1aa4155d0fc
+  sha256: ae14018c6992c6e8f415e142ee8bbcb3b0932927203fe2061a6ac87169631fef
 
 build:
   number: 0
-  skip: True # [py27]
-  skip: True # [osx]
+  skip: True # [py27 or osx]
```
Comment (epruesse) [inline recipes/tobias/meta.yaml]: thanks

**Pull request merged**